lib/config.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+var config = require('bedrock').config;
+
+config.prerender = {};
+
+// Prerending is enabled only on defined routes.
+// An optional route specific `cacheTtl` (sec) may be specified.
+config.prerender.routes = {
+  // '/about': {cacheTtl: 10000},
+  // '/i/*': {}
+};
+
+// Whitelist a single url path or multiple url paths. Compares using regex,
+// so be specific when possible. If a whitelist is supplied, only urls
+// containing a whitelist path will be prerendered. Value may be a
+// single string or an array of strings. (e.g. '/about')
+config.prerender.whitelist = null;
```

Comment:
This is a little confusing given the `routes` config option above. I get that the top one is something we added and the bottom is something we pass to the prerender library (or something like that). Can we better capture the difference in the docs here and why you'd use one over the other? If there isn't a reason to use one over the other ... we should probably hide this one.